graphql/resolvers.go

```diff
@@ -2452,28 +2452,77 @@ func (r *taskResolver) BaseTask(ctx context.Context, obj *restModel.APITask) (*r
 }
 func (r *taskResolver) ExecutionTasksFull(ctx context.Context, obj *restModel.APITask) ([]*restModel.APITask, error) {
 	i, err := obj.ToService()
+	ticket := "EVG-14623"
+	grip.Warning(message.WrapError(err, message.Fields{
+				"message":           "1",
+				"ticket":           ticket,
+				"task": 			*obj.Id,
+			}))
 	if err != nil {
 		return nil, InternalServerError.Send(ctx, fmt.Sprintf("Error getting service model for APITask %s: %s", *obj.Id, err.Error()))
 	}
+	grip.Warning(message.WrapError(err, message.Fields{
+				"message":           "2",
+				"ticket":           ticket,
+				"task": 			*obj.Id,
+			}))
 	t, ok := i.(*task.Task)
+	grip.Warning(message.WrapError(err, message.Fields{
```

Comment:
If error is nil, WrapError won't log anything, so in places where you've already validated err != nil you'll want a different if condition.